### PR TITLE
Fix typos and add design inspirations links

### LINKED ISSUE #242: [Add] SaasInterface
### URL

https://saasinterface.com/

### Description of the above provided site

Save yourself hours of research and create beautiful apps by taking inspiration from the best SaaS products.

### Category

Frontend

### Sub Category

design inspirations

### PR DESCRIPTION
**Pull request by Oyebolade**
## Fixes Issue

Closes #242 

## Screenshots
Animation typos
Fix-typos
Image typos

## Note to reviewers
Hi reviewers,
1. I added SaaS interface link to design inspirations.
2. Made a distinctive description to page collective link.
3. Fixes spelling error i.e Landingfolio link.
4. I also gave more details about land-book link.
5. Capitalize the first "it" in Kaboompics link under image subcategories.
6.

### CODE REVIEW

**Comment by Oyebolade** [comment]
I've created another branch
There is still conflict

**Review by ujjawaltyagii — COMMENTED** [review]
I would recommend resolving merge conflict first and one more thing, on deploying I can see that card's property of scaling on hover and on click property of cards isn't working ! so could you review your code 

image

(both mouse pointer on hover and card scaling on hover isn't working properly)

**Comment by rupali-codes** [comment]
Yeah, this branch is not updated with main branch

**Comment by Oyebolade** [comment]
Yes, I'm supposed to have a section with Rupali on it. I don't know how to go about the conflicts

File: database/frontend/animations.json [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+  {
+    "name": "Framer Motion",
+    "description": "Framer Motion is a production-ready motion library for React from Framer, It's simple yet powerful, allowing you to express complex user interactions with robust, semantic markup.",
+    "url": "https://www.framer.com/motion/",
+    "category": "frontend",
+    "subcategory": "animations"
+  },
+  {
+    "name": "Proton",
+    "description": "A JavaScript particle based animation engine. Which provides you tones of animated elements.",
```
Comment (CBID2) [inline database/frontend/animations.json:25]:
```suggestion
    "description": "A JavaScript particle-based animation engine that provides tons of animated elements.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Oyebolade! :) I made some suggestions to improve your descriptions! :) I hope you find them helpful! :) Keep up the good work! :)

**Comment by parma1111** [comment]
> Yeah, this branch is not updated with main branch

This PR is merged by #448 , now you may close it

**Pull request closed without merging**